### PR TITLE
Updated strong name keys to use key without a password

### LINKED ISSUE #16: Publish the private key
You should not be using a private key for strong name signing an open source project. It causes all sorts of problems, like you can't just build and run the unit tests, because you hit strong name assembly mismatch.
See doc here: https://docs.microsoft.com/en-us/dotnet/framework/app-domains/strong-named-assemblies

> If you are an open-source developer and you want the identity benefits of a strong-named assembly, consider checking in the private key associated with an assembly into your source control system.

### PR DESCRIPTION
**Pull request by dotMorten**
New password-less strong-name keys for the assemblies
Fixes #16

CC @onovotny @brianlagunas @terrajobst @karann-msft

### CODE REVIEW

**Comment by onovotny** [comment]
nit: tests don't really need a second strong name key.

Also, you can remove the need to put the key name in each project it you put it in the directory props like this:

https://github.com/dotnet/reactive/blob/master/Ix.NET/Source/Directory.build.props#L12-L13

**Comment by onovotny** [comment]
Also seems to have an extra `public.snk` key checked in?

**Comment by dotMorten** [comment]
@onovotny D'oh! Not sure what I was thinking. Fixed.

**Comment by dotMorten** [comment]
Yeah the public key was temporary and an accident. Already addressed

File: src/Microsoft.Xaml.Behaviors/AssemblyInfo.cs [hunk 1]
```diff
@@ -39,10 +39,5 @@
...
 
-#if SIGN_REAL
-[assembly: AssemblyKeyFile("FinalPublicKey.snk")]
-[assembly: AssemblyDelaySign(true)]
-[assembly: InternalsVisibleTo("UnitTests, PublicKey='002400000480000094000000060200000024000052534131000400000100010007d1fa57c4aed9f0a32e84aa0faefd0de9e8fd6aec8f87fb03766c834c99921eb23be79ad9d5dcc1dd9ad236132102900b723cf980957fc4e177108fc607774f29e8320e92ea05ece4e821c0a5efe8f1645c4c0c93c1ab99285d622caa652c1dfad63d745d6f2de5f17e5eaf0fc4963d261c8a12436518206dc093344d5ad293")]
-#else
-[assembly: InternalsVisibleTo("UnitTests")]
-#endif
\ No newline at end of file
+[assembly: AssemblyKeyFile("Behaviors.snk")]
```
Comment (onovotny) [inline src/Microsoft.Xaml.Behaviors/AssemblyInfo.cs]: Nit: no really needed with the msbuild settings.

**Pull request merged**